File: chapter 1/ex_1_3.js
const solution = (a, b, c) => {
  if (a < b && a < c) {
    return (b ** 2) + (c ** 2);
  } else if (b < a && b < c) {
    return (a ** 2) + (c ** 2);
  }
  return (a ** 2) + (b ** 2);
};

console.log(solution(3, 2, 1));

File: chapter 1/ex_1_3.erl
-module('ex_1_3').
-export([solution/3]).
-compile([debug_info]).

solution(A, B, C) ->
    if
	(A < B) and (A < C) ->
	    (math:pow(B, 2) + math:pow(C, 2));
	(B < A) and (B < C) ->
	    (math:pow(A, 2) + math:pow(C, 2));
	true ->
	    (math:pow(A, 2) + math:pow(B, 2))
    end.
